helevents/forms.py

```diff
@@ -20,23 +20,34 @@ class LocalOrganizationAdminForm(OrganizationForm):
         ),
     )
 
+    financial_admin_users = forms.ModelMultipleChoiceField(
+        get_user_model().objects.all(),
+        required=False,
+        help_text=_(
+            "Hold down “Control”, or “Command” on a Mac, to select more than one."
+        ),
```

Comment:
Might be redundant

Reply:
This one is responsible for the little gray text underneath the user selection. Without this, the info text doesn't appear like it appears for the rest of the selections. So a little detail to make the selections appear exactly the same :-)